assignment6/rearranging_cars.py

```diff
@@ -0,0 +1,72 @@
+
+class Movement(object):
+    """This class stores information about one movement,
+    i.e. a car moving from position *fro* to position *to*.
+    """
+
+    def __init__(self, car, fro, to):
+        self.car = car
+        self.fro = fro
```

Comment:
The movement class is really just a wrapper class that is used to store the movements, so that the code will be more extendable (as suggested in slack). 
(Pylint was unhappy when I tried using "from", probably because of "from __future__ import print_function" and the like XD.)